src/PrestaShopBundle/Form/Admin/Sell/Product/ProductPriceType.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ProductPriceType extends AbstractType
```

Comment:
Missing Phpdoc (please make sure you explain all your classes and all your methods)

Reply:
A good class Phpdoc describes the current state of the class, regardless of previous or future behavior. 

Consider this example: someone who will start working with PS in 2021 will don't understand (nor care) about `AdminProductController` because it was already gone when they arrived, so there's no use talking about it. Similarly, using time-bound terms like "for now" and "will soon" in the description is generally a bad idea because those statements will quickly become at best obsolete (if the predicted future becomes a reality) or worse, inaccurate (if the prediction is never fulfilled or done differently).

Therefore, I recommend using `@todo` if you need to speak about future work.

Reply:
Yes I see your point, I wrote this because I know this PHPDoc will be modified really soon But I'll use @todo next time indeed